indico/modules/rb/forms/settings.py

```diff
@@ -42,6 +42,17 @@ class SettingsForm(IndicoForm):
     notification_before_days_monthly = IntegerField(_('Send booking reminders X days before (monthly)'),
                                                     [InputRequired(), NumberRange(min=1, max=30)])
     notifications_enabled = BooleanField(_('Reminders enabled'))
+    notification_before_end_daily = IntegerField(_('Send reminders about ending bookings X days before their end date '
+                                                   '(daily)'),
```

Comment:
What about "(daily bookings)"? Because otherwise people may get the impression that those reminders are sent daily.

Reply:
(same for options below, of course...)